src/ray/rpc/worker/core_worker_client.h

```diff
@@ -53,10 +53,10 @@ namespace ray {
 namespace rpc {
 
 /// The maximum number of requests in flight per client.
-const int64_t kMaxBytesInFlight = 16 * 1024 * 1024;
+constexpr int64_t kMaxBytesInFlight = 16L * 1024 * 1024;
```

Comment:
Randomly find this PR, two things:
- We should have a human-readable unit library, something like `16_MiB`;
- `inline` is needed if we declare constants at header file
  + C++ spec doesn't guide the behavior on `constexpr`, when included by multiple translation units, whether the symbol appears once or multiple times, so it's a compiler-based UB

Reply:
Do you think it looks ok to you? https://github.com/ray-project/ray/pull/48638

Reply:
yes but let's not block this pr. we can always update later

Reply:
`inline constexpr` is needed, yes

Reply:
ya good point inlined, can integrate into library after, there's other instances of stuff like this too can go at once